### PR TITLE
Update course page

### PR DESCRIPTION
**Pull request by MatthiasReumann**
- Hidable Planned Lectures 
- Sortable VoDs
- Next Lecture Field (Notification Effect when Coming Up or Live)
- If Lecture is not coming up or live then the next lecture field date will be 'Today <time>', 'Tomorrow <time>', or '01.01.2022 <time>
- A bit of a redesign
- Changed the header and semester selection color to white in light-mode

### CODE REVIEW

File: model/stream.go [hunk 1]
```diff
@@ -108,3 +114,17 @@ func (s Stream) FriendlyDate() string {
...
+	starty, startm, startd := s.Start.Date()
+
+	if nowy == starty && nowm == startm && nowd == startd {
+		return "Today,"
+	} else if tmrwy == starty && tmrwm == startm && tmrwd == startd {
+		return "Tomorrow,"
+	}
+
+	return fmt.Sprintf("%v %02d.", s.Start.Month(), s.Start.Day())
+}
```
Comment (joschahenningsen) [inline model/stream.go]: Using jinzhu/now could improve readability: 
```suggestion
func (s Stream) FriendlyNextDate() string {
	if now.With(s.Start).EndOfDay() == now.EndOfDay() {
		return "Today,"
	}
	if now.With(s.Start).EndOfDay() == now.With(time.Now().Add(time.Hour*24)).EndOfDay() {
		return "Tomorrow,"
	}
	return s.Start.Format("Mon, January 02.")
}
```
Also, this adds the day of the week to the date.

File: web/template/planned_course_list.gohtml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+{{define "planned_course_list"}}
```
Comment (joschahenningsen) [inline web/template/planned_course_list.gohtml:1]: A placeholder for when there are no upcoming streams would be cool

File: web/template/vod_course_list.gohtml [hunk 3]
```diff
@@ -0,0 +1,82 @@
+{{define "vod_course_list"}}
```
Comment (joschahenningsen) [inline web/template/vod_course_list.gohtml:1]: And a placeholder for 'no vods yet' as well

**Review by joschahenningsen — CHANGES_REQUESTED** [review]
Love it! just a nit or two :)

File: web/template/header.gohtml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {{define "header"}}
-    <nav class="flex z-40 lg:z-50 sticky top-0 w-full items-center justify-between dark:bg-secondary bg-secondary-light p-3 h-20 z-40">
+    <nav class="flex z-40 lg:z-50 sticky top-0 w-full items-center justify-between dark:bg-secondary bg-white shadow p-3 h-20 z-40">
```
Comment (joschahenningsen) [inline web/template/header.gohtml:2]: IMO this is _very_ bright, but I don't really mind either

File: model/stream.go (hunk 1 above)
Comment (MatthiasReumann) [reply model/stream.go]: That's great. Thanks!

Comment (joschahenningsen) [reply model/stream.go]: Oh and this function is exported, it should have some documentation (shame on me for not always doing this myself :D)

File: web/template/header.gohtml (hunk 4 above)
Comment (MatthiasReumann) [reply web/template/header.gohtml:2]: I kinda like it that way. (However, I use light mode everywhere :D)

Comment (joschahenningsen) [reply web/template/header.gohtml:2]: You're the pro then 😀

**Review by joschahenningsen — APPROVED** [review]
:beers:

**Pull request merged**